Moonscraper Chart Editor/Assets/Scripts/Game/NoteVisuals/NoteVisuals3DManager.cs

```diff
@@ -163,15 +165,15 @@ Material[] GetMaterials(Material colorMat, Note.NoteType visualNoteType)
 
         switch (visualNoteType)
         {
-            case Note.NoteType.Hopo:
+            case VisualNoteType.Hopo:
                 materials = isStarpower ? resourceSharedMatsSpHopo : resourceSharedMatsHopo;
                 break;
 
-            case Note.NoteType.Tap:
+            case VisualNoteType.Tap:
                 materials = isStarpower ? resourceSharedMatsSpTap : resourceSharedMatsTap;
                 break;
 
-            case Note.NoteType.Cymbal:
+            case VisualNoteType.Cymbal:
```

Comment:
Handle case for Tom, let it fall through to default just to make things a bit clearer.